spec/forms/school_search_form_spec.rb

```diff
@@ -39,7 +39,8 @@
     it "finds schools that have a partnership" do
       school = schools[2]
       lead_provider = FactoryBot.create(:lead_provider)
-      Partnership.create!(school: school, lead_provider: lead_provider)
+      cohort = FactoryBot.create(:cohort, start_year: 2021)
```

Comment:
Don't need the `FactoryBot` here I don't think

Reply:
Currently, the partnership model requires cohort_id , because the db is cleaned every time - no cohort exist. If I remove that - the test fails. Perhaps I've misunderstood the comment - more than happy to have a chat about it

Reply:
You can just write `create`, no need for `FactoryBot`

Reply:
Done!